### ISSUE TITLE
Cannot deploy controller in 1.5.1 - open ./v1.26.1/_definitions.json: no such file or directory

### ISSUE THREAD

**Issue by fruboes** [nginxinc/nginx-ingress-helm-operator#158]
**Describe the bug**
I'm unable to deploy nginx ingress controller in openshift 4.12.22. Operator installs OK, but when creating the controller I get the following error in operator logs:

```
{"level":"error","ts":"2023-08-25T10:27:59Z","msg":"Reconciler error","controller":"nginxingress-controller","object":{"name":"nginxingress-sample","namespace":"nginx-ingress-operator-system"},"namespace":"nginx-ingress-operator-system","name":"nginxingress-sample","reconcileID":"4d7d8252-1a84-4d1b-a31b-8b3689b46f97","error":"failed to install release: values don't meet the specifications of the schema(s) in the following chart(s):\nnginx-ingress:\nopen ./v1.26.1/_definitions.json: no such file or directory","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.5/pkg/internal/controller/controller.go:329\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.5/pkg/internal/controller/controller.go:274\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.5/pkg/internal/controller/controller.go:235"}
```

and controller is not created (i.e. no controller pods are running)

**To Reproduce**
Install v1.5.1 operator (I tried both OLM and manual installs) and try creating ingress controller. It gives the error above. v1.5.0 runs ok.

**Expected behavior**
Deploy controller successfully

**Your environment**
* Version of the NGINX Ingress Operator - 1.5.1
* Kubernetes platform (e.g. Mini-kube or GCP): openshift  4.12.22
* Using NGINX or NGINX Plus: nginx

**Comment by vepatel** [comment]
@lucacome directory ref seems to be correct in error log `./v1.26.1/_definitions.json: no such file or directory"` as per https://github.com/nginxinc/nginx-ingress-helm-operator/tree/v1.5.1/helm-charts/nginx-ingress/v1.26.1 [introduced](https://github.com/nginxinc/nginx-ingress-helm-operator/pull/155) in this PR,

**Comment by vepatel** [comment]
https://github.com/nginxinc/nginx-ingress-helm-operator/pull/161#issuecomment-1697157209

**Comment by vepatel** [comment]
@fruboes fixed in v1.5.2
